packages/palette-docs/package.json

```diff
@@ -13,6 +14,9 @@
     "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
     "type-check": "tsc --pretty --noEmit"
   },
+  "workspaces": {
+    "nohoist": ["gatsby", "gatsby-*"]
```

Comment:
Was this setup by design in order to keep things cleanly separated or was there a conflict somewhere?

Reply:
The problem was that gatsby was resolving plugins relative to itself, and since gatsby doesn't know what plugins you're going to use, it can't specify them in it's package.json and yarn can't know that they must be hoisted too. so if you have a structure like this:

```
./
./node_modules/gatsby
./packages/palette-docs/gatsby-plugin-mdx
```

and then gatsby tries to resolve `gatsby-plugin-mdx`, it will fail.

See this for more deets: https://github.com/gatsbyjs/gatsby/issues/2135

Reply:
👍